README.md

```diff
@@ -55,8 +55,8 @@ overwritten during the deployment.
 2. Insert the SD card into a card reader on your host PC;
 3. Run Raspberry Pi Imager with `repo` parameter to retrieve the latest AosCore image:
 
-   ```sh
-   rpi-imager --repo https://raw.githubusercontent.com/aosedge/meta-aos-rpi/main/os_list.json
+   ```console
+   user@host:~$ rpi-imager --repo https://raw.githubusercontent.com/aosedge/meta-aos-rpi/main/os_list.json
```

Comment:
Added commands without prompt for purpose: you can just copy the whole block and past into your terminal. Otherwise you have to select part of line.

Reply:
Then, you need to be consistent and have same approach across the document which is not true
I'll adjust